### DISCUSSION TITLE
[FR] Add activity indicators to list of forks

### DISCUSSION THREAD

**Discussion by FalcoGer** [community/community#38195] [Other Feature Feedback, Questions, & Ideas]
# The problem

Often a project is abandoned by the original people who made a repository and then other people pick the project up in a fork, or a project is forked and the fork itself becomming it's own, major then. If it is a reasonably large project, a great number of forks may exist and it is hard to find which ones are actively being worked on and which are a one off pull request that are just cluttering up the list.

Right now the best option is to scroll through the list and try and find forks which have been forked themselves or simply click every single one and check the status there, which is sub-optimal.

# The solution

Add some indicators on how a fork is doing, when the last commit was and how many commits ahead/behind a project is in regards to it's parent, how many stars it got, and perhaps allow us to sort by those metrics.

If we see then see that a fork is 83 commits behind the parent or hasn't been committed to for 2 years, then we can ignore it.

**Comment by david-engelmann** [comment]
This addition would definitely improve the insights->fork experience

**Comment by ossanna16** [comment]
Hi there @FalcoGer and welcome to our community! Thank you for the feedback 🙂

**Comment by ethanpil** [comment]
+1000 this will be super useful for finding forks that add value to a project!

**Comment by SirYodaJedi** [comment]
Something like this (but prettier and cleaner) could maybe work, to indicate commits ahead, commits behind, stars, and people watching, respectively.
> 🖼️ [UserName](url) / [repo_name](url) `➕100` `➖30` `⭐6.2K` `👁️378`

**Reply by FalcoGer** [reply]
I think a last commit date would also be really helpful.

**Reply by jsskeen-utah** [reply]
yes, so you can see if that fork is dead as well

**Comment by jsskeen-utah** [comment]
I came to GitHub feedback to add this exact suggestion! This could really help open-source projects not die!

**Reply by FalcoGer** [reply]
it is a thing now, and has been for a while.
